### PR TITLE
feat: add support to specify secret accessors

### PR DESCRIPTION
**Pull request by ps-occrp**
This PR supports optionally adding list of secret accessors who will be able access specific secret being created.

### CODE REVIEW

**Comment by ps-occrp** [comment]
@apeabody can you please review this PR?

File: variables.tf [hunk 1]
```diff
@@ -54,3 +54,9 @@ variable "add_pubsub_permissions" {
   description = "The list of the pubsub topics to give secret manager access to"
   default     = []
 }
+
+variable "secret_accessors_list" {
+  type        = list(string)
+  description = "The list of the members to allow accessing secret"
```
Comment (apeabody) [inline variables.tf:60]: nit: plural?
```suggestion
  description = "The list of the members to allow accessing secrets"
```

File: main.tf [hunk 2]
```diff
@@ -88,3 +88,12 @@ resource "google_secret_manager_secret_version" "secret-version" {
   secret      = google_secret_manager_secret.secrets[each.value.name].id
   secret_data = each.value.secret_data
 }
+
+resource "google_secret_manager_secret_iam_binding" "binding" {
+  depends_on = [google_secret_manager_secret.secrets]
+  project    = var.project_id
+  for_each   = { for secret in var.secrets : secret.name => secret }
```
Comment (apeabody) [inline main.tf:95]: Would it be possible to `for_each` on `google_secret_manager_secret.secrets`?  That would remove the need for the `depends_on` which is prefferable.

**Review by apeabody — CHANGES_REQUESTED** [review]
Thanks for the contribution @ps-occrp!  A few suggestions

File: main.tf (hunk 2 above)
Comment (ps-occrp) [reply main.tf]: Updated it

**Review by apeabody — APPROVED** [review]

**Review by apeabody — APPROVED** [review]
Thanks for the contribution @ps-occrp!

**Pull request merged**